almond/src/main/scala/plotly/Almond.scala

```diff
@@ -4,6 +4,7 @@ import java.lang.{Boolean => JBoolean, Double => JDouble, Integer => JInt}
 
 import almond.interpreter.api.{DisplayData, OutputHandler}
 
+import scala.collection.immutable.Seq
```

Comment:
Let's keep relying on the default `Seq`. That makes the API slightly different in 2.12 and 2.13, but that these APIs accept `Seq(…)` in both cases, which is not the case in 2.12 if we require `scala.collection.immutable.Seq` explicitly.